app/jobs/parity_check_job.rb

```diff
@@ -1,5 +1,5 @@
 class ParityCheckJob < ApplicationJob
-  queue_as :high_priority
+  queue_as :migration
```

Comment:
I don't think this needs to move to migration 🤔

Reply:
changed it back